### PR TITLE
Drop windows patch

### PR DESCRIPTION
**Pull request by jsignell**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

https://github.com/conda-forge/graphviz-feedstock/pull/59
https://github.com/dask/dask/issues/7231

@jakirkham

### CODE REVIEW

**Comment by jsignell** [comment]
@conda-forge-admin, please rerender

**Comment by jakirkham** [comment]
Thanks Julia! 😄

Could we also add a version requirement to `graphviz` to ensure it is `2.46.1` or greater? Want to make sure we are only using `graphviz` packages with the changes in PR ( https://github.com/conda-forge/graphviz-feedstock/pull/59 )

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,18 +11,8 @@ source:
...
-  script:
-    # Needed due to the Windows batch script wrappers in conda-forge Graphviz.
-    # see https://github.com/conda-forge/graphviz-feedstock/blob/bbfe3c7be2a448dd11db165a18c3edfd5ee6a95d/recipe/bld.bat#L27-L32
-    # Must patch manually as the source code and patch are CRLF and conda-build
-    # tries to convert the patch to LF, which then won't apply.
-    - pushd "{{ RECIPE_DIR }}"
-    - cp windows-bat.patch "{{ SRC_DIR }}"
-    - popd
-    - patch --binary -p0 -i windows-bat.patch
-    - python -m pip install --no-deps --ignore-installed "${SRC_DIR}"
```
Comment (jakirkham) [inline recipe/meta.yaml:25]: Also would keep `script` and this last line to make sure it is still being installed

Comment (jsignell) [reply recipe/meta.yaml:25]: Ooops! Got over-zealous in the cleanup.

**Review by jsignell — COMMENTED** [review]
:heart:

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks Julia! 😄

**Comment by jakirkham** [comment]
cc @xflr6 (for awareness)

**Pull request merged**